<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurrentSeedSearch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>456</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>実数値から</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>205</y>
        <width>291</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Label_Frame">
         <property name="text">
          <string>個体値乱数列の消費数の範囲: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SB_FirstFrame"/>
       </item>
       <item>
        <widget class="QSpinBox" name="SB_LastFrame"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>10</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="Label_Level">
         <property name="text">
          <string>レベル</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="SB_Level">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>171</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="Label_PokemonName">
         <property name="text">
          <string>種族</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="CB_PokemonName">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_NatureName">
         <property name="text">
          <string>性格</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="CB_NatureName">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>75</y>
        <width>296</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsH">
           <property name="text">
            <string>HP  </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsH">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsA">
           <property name="text">
            <string>攻撃</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsA">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_10" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsB">
           <property name="text">
            <string>防御</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsB">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_7" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsC">
           <property name="text">
            <string>特攻</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsC">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsD">
           <property name="text">
            <string>特防</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsD">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="Label_StatsS">
           <property name="text">
            <string>素早</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SB_StatsS">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>レポ針から</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_DateTime">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>386</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>日時</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>252</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="GL_FirstDateTime">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_20">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SB_Year"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Label_Year">
        <property name="text">
         <string>年</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="SB_Month">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="Label_Month">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="SB_Day">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="Label_Day">
        <property name="text">
         <string>日</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>65</y>
       <width>228</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="SB_Hour">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Label_Hour">
        <property name="text">
         <string>時</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="SB_Minute">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="Label_Minute">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="SB_Second">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="Label_Second">
        <property name="text">
         <string>秒</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>65</y>
       <width>82</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_PlusMinus">
        <property name="text">
         <string>±</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SB_SecondRange">
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Label_SecondRange">
        <property name="text">
         <string>秒</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_4</zorder>
   </widget>
   <widget class="QTextEdit" name="TE_Result">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>35</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_EXEC">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>実行</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Result">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>15</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>結果</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
